app/celery/service_callback_tasks.py

```diff
@@ -191,8 +203,14 @@ def send_inbound_sms_to_service(self, inbound_sms_id, service_id):
         raise e
 
 
-def create_delivery_status_callback_data(notification, service_callback_api):
+def create_delivery_status_callback_data(notification, service_callback_api, provider_payload=None):
```

Comment:
Annotations + docstring

Reply:
Also, `provider_payload={}`, then remove the two lines to check and set it to an empty dictionary.